src/components/common/CopyButton/index.tsx

```diff
@@ -8,8 +8,15 @@ import CopyToClipboard from 'react-copy-to-clipboard'
 import { DISPLAY_TEXT_COPIED_CHECK } from 'apps/explorer/const'
 
 const Icon = styled(FontAwesomeIcon)<{ copied: boolean }>`
-  color: ${({ theme, copied }): string => (copied ? theme.green1 : theme.icon)};
+  color: ${({ theme, copied }): string => (copied ? theme.green : theme.icon)};
   cursor: ${({ copied }): string => (copied ? 'reset' : 'pointer')};
+  transition: color 0.2s ease-in-out, opacity 0.2s ease-in-out;
```

Comment:
Provides a transition effect for the changing icon/color and also styles the 'copied' text.

Reply:
I'm ok with the transition, although I thought it was creating a weird effect

Reply:
Glad I didn't even try this :D

Reply:
Now I saw the effect.
Agree, the icon color transition feels weird...

Reply:
Thanks for the feedback :D - think it's good to remove, will address in this PR.

Reply:
Removed.